gbfs.md

```diff
@@ -571,15 +572,89 @@ Describes pricing for the system.
 \-&nbsp;`plan_id` | Yes | ID | Identifier for a pricing plan in the system.
+\-&nbsp;`vehicle_type_id` | Optional | Array | `vehicle_type_id` of the vehicle eligible for this pricing plan as described in [vehicle_types.json](#vehicle_typesjson-added-in-v21-rc).<br /><br />If this field is empty, the plan applies to all vehicle types defined in the dataset.<br /><br />In the case of a vehicle type being attributed to different plans, all plans associated with the vehicle type are valid.   
 \-&nbsp;`url` | Optional | URL | URL where the customer can learn more about this pricing plan.
 \-&nbsp;`name` | Yes | String | Name of this pricing plan.
 \-&nbsp;`currency` | Yes | String | Currency used to pay the fare. <br /><br /> This pricing is in ISO 4217 code: http://en.wikipedia.org/wiki/ISO_4217 <br />(e.g. `CAD` for Canadian dollars, `EUR` for euros, or `JPY` for Japanese yen.)
-\-&nbsp;`price` | Yes | Non-negative float OR String | Fare price, in the unit specified by currency. If String, must be in decimal monetary value.
+\-&nbsp;`price` | Yes | Non-negative float OR String | Fare price, in the unit specified by currency. If String, must be in decimal monetary value.<br /><br />In case of non-variable price, this field is the total price. In case of variable price, this field is the base price that is charged only once per trip (e.g., price for unlocking). See `variable_price` for details.
 \-&nbsp;`is_taxable` | Yes | Boolean | Will additional tax be added to the base price?<br /><br />`true` - Yes.<br />  `false` - No.  <br /><br />`false` may be used to indicate that tax is not charged or that tax is included in the base price.
 \-&nbsp;`description` | Yes | String | Customer-readable description of the pricing plan. This should include the duration, price, conditions, etc. that the publisher would like users to see.
+\-&nbsp;`per_km_pricing` | Optional | Array | Array of segments when the price is a function of distance travelled, displayed in kilometers.<br /><br />If this array is not provided, there are no variable prices based on distance.
+&emsp;&emsp;\-&nbsp;`start` | Yes | Non-Negative Integer | Number of kilometers that have to elapse before this segment starts applying.
+&emsp;&emsp;\-&nbsp;`rate` | Yes | Float | Rate that is charged after the `start`. Can be a negative number, which indicates that the traveller will receive a discount.
+&emsp;&emsp;\-&nbsp;`interval` | Yes | Non-Negative Integer | Interval in kilometers at which the `rate` of this segment is reapplied indefinitely, unless defined by `end`.<br /><br />An interval of 0 indicates the rate is only charged once.
+&emsp;&emsp;\-&nbsp; `end` | Optional | Non-Negative Integer | Number of kilometers that have to elapse after this segment has started before this segment stops applying.<br /><br /> If this field is empty, the price issued form this segment is charged until the trip ends, in addition to following segments.
```

Comment:
Nit: typo "form" -> "for"

Ah, can you confirm that per-description this is not "The kilometer at which the rate should stop applying" but rather "The number of kilometers that have to be travelled from start for this rate to stop applying"? If that's the case, the name of the field should be different. Perhaps `duration`? Or maybe just change the description for this to be "The kilometer after which the rate will no longer apply" (I think this is clearer).

Reply:
Updated the description to your suggestions of "The kilometer after which the rate will no longer apply". Thanks for the suggestions and all of your comments!